src/lib/learn-client/api/api-types.ts

```diff
@@ -75,6 +76,24 @@ export interface ApiCollectionTutorial
 	tutorial: ApiTutorial
 }
 
+export interface ApiCollectionTutorialProgress
+	extends Pick<BaseModel, 'id' | 'created_at' | 'updated_at'> {
```

Comment:
Since this is `Pick`-ing all the attributes, can we just extend `BaseModel` here?

Reply:
💖 Ah great call, I'd mindlessly copied from somewhere else 😁 ✅ updated!